packages/ui/src/components/StoryPublicationDate/StoryPublicationDate.tsx

```diff
@@ -0,0 +1,19 @@
+import type { Culture, Story } from '@prezly/sdk';
+import { type AlgoliaStory, getStoryPublicationDate } from '@prezly/theme-kit-core';
+
+interface Props {
+    story: Story | AlgoliaStory;
+    locale?: Culture['code'];
+}
+
+export function StoryPublicationDate({ story, locale }: Props) {
+    const date = getStoryPublicationDate(story);
+
+    if (!date) {
+        return null;
+    }
+
+    return (
+        <>{date.toLocaleDateString(locale, { year: 'numeric', month: 'long', day: 'numeric' })}</>
+    );
```

Comment:
I believe date formats should be customizable. 

We have this setting available in the app:

![image](https://github.com/prezly/theme-kit-js/assets/370680/0d84b1a5-f2d3-498a-a1f7-dd7546053eda)

Reply:
Updated https://github.com/prezly/theme-kit-js/pull/414/commits/7f5e4071f2ebd39b7cae32883fec555728b1ac8b